rcode-reader/src/main/java/eu/rssw/pct/PLReader.java

```diff
@@ -0,0 +1,198 @@
+/**
+ * Copyright 2005-2017 Riverside Software
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+package eu.rssw.pct;
+
+import java.io.ByteArrayInputStream;
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.RandomAccessFile;
+import java.nio.ByteBuffer;
+import java.nio.channels.FileChannel;
+import java.nio.charset.Charset;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Class for reading and extracting contents of a Progress Library file.
+ * 
+ * @author <a href="mailto:g.querret+PCT@gmail.com">Gilles QUERRET</a>
+ */
+public class PLReader {
+    private static final int MAGIC = 0xd707;
+    private static final int MAGIC_V11 = 0xd70b;
+    private static final int MAGIC_V11_MM = 0xd70c;
+//    private static final int MAGIC_SIZE = 2;
+    private static final int ENCODING_OFFSET = 0x02;
+    private static final int ENCODING_SIZE = 20;
+    private static final int FILE_LIST_OFFSET = 0x1e;
+    private static final int FILE_LIST_OFFSET_V11 = 0x22;
+    // private static final int RECORD_MIN_SIZE = 29;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=squid%3ACommentedOutCodeLine)